lib/bpk-component-text/src/BpkText.js

```diff
@@ -97,6 +97,10 @@ const TEXT_TOKENS = {
 };
 
 const getStyleByTextStyle = textStyle => {
+  if (textStyle === 'inherit') {
```

Comment:
Let's remove this from here and just return an empty object in the mapping bellow.

Reply:
Changed